### ISSUE TITLE
--prikey option should be removed

### ISSUE THREAD

**Issue by Lvceo** [vishalxl/nostr_console#34]
--prikey option should be removed as it's a security breach, IMHO.

What about having this shown to every user logged into the system?
```
$ ps ax | grep nostr
 553521 pts/1    Sl+    0:02 ./nostr_console_elf64 --prikey 680a0693d62e04198472d6def92b801d11fbd417f223000d45f4bbb1e6a3a851
```

Private key should be imperatively passed over a dotfile in HOME DIR.

**Comment by vishalxl** [comment]
good point. what about the idea of having a password encrypted dotfile? Would this be needed for example on windows ? Linux has file permissions, but do we still want files containing private keys to be encrypted by some other password?

**Comment by jblachly** [comment]
> dotfile should not be encrypted

Concur 100%; dotfiles even with authorization credentails (example of .ssh config given, but also e.g. AWS credentials file) are just mode-protected

**Comment by vishalxl** [comment]
Very delayed reply, but I just fixed so that Nostr Console will read private key from a config file. The config file is a json config file that can be created using Nostr Commander ( or can be created manually). More details here: https://github.com/vishalxl/nostr_console/discussions/89
